labelcolourselector.py

```diff
@@ -3,8 +3,11 @@ def __init__(self, project):
         self._project = project
 
     def get_colour(self, label):
-        if (label in ('jira-type:task', 'jira-type:story', 'Task', 'Story', 'story')): return '7bc043'
-        elif (label == 'Bug'): return 'ee4035'
+        if label in ('rfe', 'story'):
+            return '7bc043'
+        elif label == 'bug':
+            return 'ee4035'
```

Comment:
```suggestion
        elif label == 'bug':
            return 'ee4035'
        elif label == 'epic':
            return 'ddf4dd'
```
Last suggestion, WDYT of a different light color <img width="16" alt="image" src="https://user-images.githubusercontent.com/91831478/174745872-86555ae8-110e-4d4a-84a9-6c9bd3d27ea8.png"> for "epic" label?

Reply:
Ready to merge otherwise 🙂

Reply:
don't care 😄, planned to tweak colours after migration, happy to commit though